ovirt-ansible-roles.spec.in

```diff
@@ -18,6 +18,7 @@ BuildArch:      noarch
 Url:            http://www.ovirt.org
 
 Requires: ansible >= 2.3.0
+Requires: python2-jmespath
```

Comment:
This will break on Fedora, where package is named python-jsmespath. Please add condition to differentiate between Fedora and CentOS

Reply:
On my fedora it has same name, let me re-check.

Reply:
https://koji.fedoraproject.org/koji/packageinfo?packageID=18913

Reply:
```bash
$ rpm -q --provides python2-jmespath
python-jmespath = 0.9.0-7.fc26
python2-jmespath = 0.9.0-7.fc26
```

https://koji.fedoraproject.org/koji/rpminfo?rpmID=9077526

Reply:
Ahh, sorry, I missed that :-(

Reply:
$ rpm -qi python-jmespath
package python-jmespath is not installed

$ rpm -qi python2-jmespath
Name        : python2-jmespath
Version     : 0.9.0
Release     : 7.fc26
Architecture: noarch
Install Date: Tue 29 Aug 2017 10:22:58 AM CEST
Group       : Unspecified
Size        : 133359
License     : MIT
Signature   : RSA/SHA256, Sun 12 Feb 2017 12:33:53 PM CET, Key ID 812a6b4b64dab85d
Source RPM  : python-jmespath-0.9.0-7.fc26.src.rpm
Build Date  : Sat 11 Feb 2017 08:30:42 PM CET
Build Host  : buildvm-17.phx2.fedoraproject.org
Relocations : (not relocatable)
Packager    : Fedora Project
Vendor      : Fedora Project
URL         : https://github.com/jmespath/jmespath.py
Summary     : JSON Matching Expressions
Description :
JMESPath allows you to declaratively specify how to extract elements from
a JSON document.